elements/pf-tabs/pf-tab.ts

```diff
@@ -68,14 +75,77 @@ import styles from './pf-tab.css';
  * @fires { TabExpandEvent } tab-expand - when a tab expands
  */
 @customElement('pf-tab')
-export class PfTab extends BaseTab {
-  static readonly styles = [...BaseTab.styles, styles];
+export class PfTab extends LitElement {
+  static readonly styles = [BaseStyles, styles];
+
+  static override shadowRootOptions: ShadowRootInit = { ...LitElement.shadowRootOptions, delegatesFocus: true };
+
+  @query('button') private button!: HTMLButtonElement;
+
+  @queryAssignedElements({ slot: 'icon', flatten: true })
+  private icons!: Array<HTMLElement>;
+
+  @property({ type: Boolean }) manual = false;
 
   @observed
   @property({ reflect: true, type: Boolean }) active = false;
 
-  @observed
```

Comment:
Pretty sure I need to add this back in to inform RTI to rebuild, need to add an event that TabsController listens for.